configure.ac

```diff
@@ -324,6 +324,7 @@ if test "$enable_man" = "yes"; then
 	AC_PATH_PROG([XSLTPROC], [xsltproc])
 	if test -z "$XSLTPROC"; then
 		enable_man=no
+		AC_MSG_WARN(xsltproc is missing. Turning off --enable-man.)
```

Comment:
should be quoted: `AC_MSG_WARN([xsltproc is ..........])`

Reply:
Thanks